build_tools/benchmarks/convperf/build_and_run_convperf.sh

```diff
@@ -47,7 +47,10 @@ popd # external/iree
 popd # ${REPO_DIR}
 
 # Build ConvPerf.
-cmake -GNinja -B "${BUILD_DIR}" "${REPO_DIR}"
+cmake -GNinja \
+    -DCMAKE_CXX_COMPILER=/usr/bin/clang++ \
+    -DCMAKE_C_COMPILER=/usr/bin/clang \
```

Comment:
This hardcodes a specific path at which to find clang, which isn't great. Better would be `"$(which clang)"` or I think just setting it as "clang" would make CMake do the same lookup. Generally though, we encode this in `CC` and `CXX` env variables. Those are set in the Docker containers or, e.g. https://github.com/iree-org/iree/blob/ceb9cfd8c955f8fed500112702c7f7cafc2fc38d/build_tools/docker/dockerfiles/base.Dockerfile#L34 or the command line when they need to vary e.g. https://github.com/iree-org/iree/blob/ceb9cfd8c955f8fed500112702c7f7cafc2fc38d/.github/workflows/ci.yml#L677

Reply:
I added environment variables to the DockerFile.